test/commons/aria/implicit-role.js

```diff
@@ -270,20 +285,57 @@ describe('aria.implicitRole', function() {
 		assert.equal(implicitRole(node), 'combobox');
 	});
 
-	it('should return region for "section" with accessible name', function() {
+	it('should return region for "section" with accessible name aria-label', function() {
 		fixture.innerHTML = '<section id="target" aria-label="foo"></section>';
 		var node = fixture.querySelector('#target');
 		flatTreeSetup(fixture);
 		assert.equal(implicitRole(node), 'region');
 	});
 
+	it('should return region for section with accessible name aria-labelledby', function() {
+		fixture.innerHTML =
+			'<div id="foo">foo</div><section id="target" aria-labelledby="foo"></section>';
+		var node = fixture.querySelector('#target');
+		flatTreeSetup(fixture);
+		assert.equal(implicitRole(node), 'region');
+	});
+
+	it('should return region for section with accessible name title', function() {
+		fixture.innerHTML = '<section id="target" title="foo"></section>';
+		var node = fixture.querySelector('#target');
+		flatTreeSetup(fixture);
+		assert.equal(implicitRole(node), 'region');
```

Comment:
This didn't work in Safari, right? If so I think we probably don't want to do that